src/v1/combo-chart/combo-chart.ts

```diff
@@ -0,0 +1,669 @@
+            display: allowLabels ? "auto" : false,
+            color: "blue",
+            textStrokeColor: "white",
+            textStrokeWidth: 5,
+            labels: {
+              title: {
+                font: {
+                  weight: "bold",
+                },
+              },
+              value: {
+                color: "black",
+              },
+            },
+          },
+          legend: {
+            display: false,
+          },
+        },
+
+        // responsive: true,
+        maintainAspectRatio: false,
+        interaction: {
+          mode: "point",
+          intersect: true,
+        },
+        onClick: (e: any) => {
+          const activeElement = e.chart.getActiveElements()[0];
+          if(!activeElement) {
+            ctx.emitEvent(ChartToTSEvent.CloseContextMenu);
+            return;
+          }
+          const dataX = activeElement?.index;
+          let {
+            type: chartType,
+            stack,
+            yPos,
+          } = activeElement?.element.$datalabels[0].$context.dataset;
+          const dataY = yPos;
+          if (stack.includes("stack")) {
+            chartType = "bar-stack";
+          }
+          logger.info(
+            "ChartPoint",
+            dataX,
+            dataY,
+            dataModel.getPointDetails(dataX, dataY)
+          );
+          ctx.emitEvent(ChartToTSEvent.OpenContextMenu, {
+            event: getParsedEvent(e),
+            clickedPoint: {
+              tuple: dataModel.getPointDetails(dataX, dataY, chartType),
+            },
+          });
+        },
+      },
+
+      plugins: [htmlLegendPlugin],
+    });
+  } catch (e) {
+    logger.error("renderfailed", e);
+    throw e;
+  }
+}
+
+const renderChart = async (ctx: CustomChartContext): Promise<void> => {
+  if (globalChartReference) {
+    globalChartReference.destroy();
+  }
+  try {
+    ctx.emitEvent(ChartToTSEvent.RenderStart);
+    render(ctx);
+  } catch (e) {
+    ctx.emitEvent(ChartToTSEvent.RenderError, {
+      hasError: true,
+      error: e,
+    });
+  } finally {
+    ctx.emitEvent(ChartToTSEvent.RenderComplete);
+  }
+};
+
+(async () => {
+  // populate it for multiple configs, need to populate common x-axis config
+  const ctx = await getChartContext({
+    getDefaultChartConfig: (chartModel: ChartModel): ChartConfig[] => {
+      const cols = chartModel.columns;
+
+      const measureColumns = _.filter(
+        cols,
+        (col) => col.type === ColumnType.MEASURE
+      );
+
+      const attributeColumns = _.filter(
+        cols,
+        (col) => col.type === ColumnType.ATTRIBUTE
+      );
+
+      const axisConfig: ChartConfig[] = [
+        {
+          key: "column",
+          dimensions: [
+            {
+              key: "x",
+              columns: [attributeColumns[0]],
+            },
+            {
+              key: "y",
+              columns: measureColumns.slice(0, 2),
+            },
+          ],
+        },
+        {
```

Comment:
we can avoid default chart for all the other chart types... no point in plotting same values again.

Reply:
updated